README.md

```diff
@@ -57,4 +53,4 @@ Now you should see the developer tools and a `<div>`-element. The number in the
 
 Copy this ID (here: 206895961) into your script.
 
-Alternatively you can use the [INWX API](https://www.inwx.de/de/help/apidoc/f/ch02s09.html#nameserver.info) to obtain the record id.
\ No newline at end of file
+Alternatively you can use the [INWX API](https://www.inwx.de/de/help/apidoc/f/ch02s09.html#nameserver.info) to obtain the record id.
```

Comment:
Okay I don't know why github (resp. the github editor window) changed here something. I think it changes just the last lineending. If you want I'll fix this.